compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUInterfaces.td

```diff
@@ -109,26 +109,6 @@ def IREEGPU_MmaInterfaceAttr : AttrInterface<"MmaInterfaceAttr"> {
         return failure();
       }]
     >,
-    InterfaceMethod<
-      /*desc=*/[{
-        Populates the reassociation indices and result shape to materialize the
-        thread layout shape from the subgroup block shape.
-      }],
-      /*retTy=*/"::mlir::LogicalResult",
-      /*methodName=*/"materializeOperandConcreteShape",
```

Comment:
This one is used though?

Reply:
Gah, this got reimplemented outside the interface at some point. I missed that, but it's wrong to drop this, we need it.

Reply:
It isn't

There's a static implementation that depends on other interface methods

Reply:
Yeah, that shouldn't have happened. It doesn't depend on other interface methods, it's hard coded for MMAAttr. If it's already like that it's still dead code though...

We can drop this but keep it mind that you'll end up needing the same thing for scaled MFMA.

Reply:
Hm ... yeah

I was going to add a `getUndistributedTileTypes(SmallVectorImpl<VectorType>&>` to generalize the `getMNKShape()` bit, but we'll probably want to have some other widget for outer types.

I figure that'll grow organically as we realize we need to generalize that code